orchestration/dagster_orchestration/hca_orchestration/tests/solids/test_create_snapshot.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from unittest.mock import patch
+import uuid
+
+from dagster import ModeDefinition, execute_solid
+from data_repo_client import SnapshotModel
+
+from hca_manage.manage import JobId
+from hca_orchestration.resources.data_repo import noop_data_repo_client
+from hca_orchestration.resources.sam import noop_sam_client
+from hca_orchestration.solids.create_snapshot import make_snapshot_public, submit_snapshot_job
+
+
+class CreateSnapshotSolidsTestCase(unittest.TestCase):
```

Comment:
We should get a test case around the snapshot failure mode I mentioned in another comment.

Reply:
There's now a test in the "wait for job completion" solid to verify that it correctly fails the solid if the job it's polling for fails.